platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -46,7 +46,8 @@ public static synchronized Mapbox getInstance(@NonNull Context context, @NonNull
       Context appContext = context.getApplicationContext();
       INSTANCE = new Mapbox(appContext, accessToken);
 
-      Events.initialize();
+      Telemetry.initialize();
+      Telemetry.updateDebugLoggingEnabled(true);
```

Comment:
Are we enabling logging by default?

Reply:
this should have gone into the testapp not the sdk

Reply:
What do you mean? This is the SDK 🤔